.github/workflows/actions.yml

```diff
@@ -797,14 +797,14 @@ jobs:
           name: binaries-${{ github.job }}
           path: pfx/**/*
 
-  debian-11-from-tarball:
+  debian-12-from-tarball:
     needs: [ make-source-tarball, what-to-make ]
     if: ${{ needs.what-to-make.outputs.make-cli == 'true' || needs.what-to-make.outputs.make-daemon == 'true' || needs.what-to-make.outputs.make-gtk == 'true' || needs.what-to-make.outputs.make-qt == 'true' || needs.what-to-make.outputs.make-tests == 'true' || needs.what-to-make.outputs.make-utils == 'true' }}
     runs-on: ubuntu-22.04
     env:
       NODE_PATH: /usr/lib/nodejs:/usr/share/nodejs
     container:
-      image: debian:11-slim
```

Comment:
question: we don't need to support debian 11 anymore?

Reply:
I've mentioned this in the commit messages, but I'll repeat it here:

The minimum `npm` version specified in CMakeLists.txt is higher than the version available in debian 11's package repository. So if we keep debian 11 support, then we will have to lower the minimum version instead.

Reply:
I'm fine with either bumping to debian 12 or lowering the minimum npm version to `7.5.2`.

Deferring the decision to @ckerr.

Reply:
That version of NPM was set nearly 2 years ago in #7219. And it's only starting to fail now?

Reply:
Question: can't we just update the installed version of NPM in debian 11? Something like `npm install npm -g`?

Reply:
Yes, that's also an option... But I don't want to do that.

It may be just paranoia, but I've always felt iffy messing with files installed by the OS package manager, dreading that it will mess up the system in unimaginable ways.

Even if it is just for a CI that will be rolled back after every run, I don't want to risk suggesting to people that this is how we recommend running `REBUILD_WEB=ON` on debian 11.

Reply:
The alternative would be using https://github.com/actions/setup-node to install the latest nodejs version, I'll be ok with that.

Reply:
I do suggest to go with [actions/setup-node](https://github.com/actions/setup-node)
Also: https://github.com/transmission/transmission/pull/7209#issuecomment-2448241063

Reply:
As debian user I'd say it's good to move from 11 to 12 at some point.
11's EOL date was 2024-08-14. There's only LTS at this point for 11.

Reply:
Updating npm like @Coeur stated is very common. I have had way better experiences with updating npm than with one of its Python counterparts. With Python I do rather want to keep the OS-provided packages than mess with PyPi versions.

Reply:
If Debian 11 is EOL, I'm fine with dropping it here

Reply:
It's not EOL, it's LTM.

Reply:
Going a bit off-topic, but debian LTS Team support starts when the release goes EOL. [https://wiki.debian.org/DebianReleases](url)